### ISSUE TITLE
Different results with find vs findy

### ISSUE THREAD

**Issue by oyeanuj** [wbotelhos/idy#3]
Hi again @wbotelhos! As I was playing with the library, I noticed a weird mismatch of results when using find and findy:

```sh
10] pry(main)> Individual.find('2R')
  Individual Load (0.5ms)  SELECT  "individuals".* FROM "individuals" WHERE "individuals"."id" = $1 LIMIT $2  [["id", 2], ["LIMIT", 1]]
=> #<Individual:0x00007f9b2be7d6c8
 id: 2,
 first_name: "The Aga",
 last_name: nil,
 created_at: Mon, 05 Feb 2018 19:41:12 PST -08:00,
 updated_at: Mon, 05 Feb 2018 19:41:12 PST -08:00>

[14] pry(main)> Individual.findy('2R')
  Individual Load (2.0ms)  SELECT  "individuals".* FROM "individuals" WHERE "individuals"."id" = $1 LIMIT $2  [["id", 1], ["LIMIT", 1]]
=> #<Individual:0x00007f9b31ace530
 id: 1,
 first_name: "First",
 last_name: "Person",
 created_at: Mon, 25 Dec 2017 08:34:34 PST -08:00,
 updated_at: Mon, 25 Dec 2017 08:34:34 PST -08:00>
```

According to the README, since the Individual model is decorated, I'd expect the same result - is my understanding correct?

**Comment by wbotelhos** [comment]
Hi @oyeanuj , thanks again for the observations.

The original method `find` ignores the letter by default, so the `'2R'` will be recognized as `'2'`.
The method `findy`, from idy gem, will read it as a string with no trick, so the `'2R'` will keeped as `'2R'` that will be translated to ID and became `1` as your example.

The trick here is original from the AR. This is the main cause of I avoid to override the original AR methods, and keep the compatibility, and to use a custom and safe method like `findy`.

**Comment by oyeanuj** [comment]
@wbotelhos Got it, thanks for the explanation. Should we clarify that in the README because the following section of the README suggests that one could *also* use `find`?

> Since you add the idy callback to your model, find method will be decorated:
> ```Article.find('My').id```
> ```# 1```

**Comment by wbotelhos** [comment]
Ops... I will update that and other things, given support to older Rails like 4.2.x.

Thanks for warn me. (:

**Comment by oyeanuj** [comment]
Perfect, that clarifies it for me - thank you!

TL;DR for anyone who stumbles upon this: use the `findy` method instead of `find`.
